src/python/pants/engine/console.py

```diff
@@ -86,7 +86,7 @@ def print_stdout(self, payload, end="\n"):
         print(payload, file=self.stdout, end=end)
```

Comment:
Might be nice to move some of the description of this issue into a comment here that explains why these are different and/or align them?